app/views/calculators/calculator.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 class="display-4 text-center mt-5 mb-5">
```

Comment:
rename this template according to controller action name

Reply:
Thanks, done.